lib/ansible/utils/module_docs_fragments/cnos.py

```diff
@@ -80,6 +80,6 @@ class ModuleDocFragment(object):
             - This specifies the type of device where the method is executed.
         required: Yes
         default: null
-        choices: [g8272_cnos,g8296_cnos,g8332_cnos]
+        choices: [g8272_cnos,g8296_cnos,g8332_cnos,NE1072T_cnos,NE1032_cnos,NE1032T_cnos,NE10032_cnos,NE2572_cnos]
```

Comment:
Created a new pull Request for this change
https://github.com/ansible/ansible/pull/26779